ludwig/cli.py

```diff
@@ -191,6 +192,11 @@ def upload(self):
 
         upload.cli(sys.argv[2:])
 
+    def pretrained_summary(self):
+        from ludwig.utils import pretrained_summary
+
+        pretrained_summary.cli_summarize_pretrained(sys.argv[2:])
+
```

Comment:
it might be good to add in the docs some example runs and outputs

Reply:
@ethanreidel To second @skanjila -- would it be possible to show an example of running this command in terms of how it is different from the existing one -- and an example output.  Thank you very much.

Reply:
Certainly.

Reply:
Qq: when you say you'd like an example, do you mean an example in the Ludwig docs or how would you prefer it?

Reply:
@ethanreidel One option is to create an example in the `examples/` top level directory in Ludwig